src/main/java/org/infoshareacademy/javaholics/AllRoute/CreateRoute.java

```diff
@@ -0,0 +1,96 @@
+package org.infoshareacademy.javaholics.AllRoute;
+import org.infoshareacademy.javaholics.Route;
+import org.infoshareacademy.javaholics.RouteDifficulty;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.util.Scanner;
+import java.util.Date;
+
+public class CreateRoute {
+    boolean status;
+    Date date = new Date();
+    Route route = new Route();
+    public Date getDate() {
+        return date;
+    }
+
+    Scanner scanner = new Scanner(System.in);
+    public void routeLength (){
+        System.out.println("Podaj początek trasy : ");
+        route.setPlaceStart(scanner.nextLine());
+        System.out.println("Podaj koniec trasy : ");
+        route.setPlaceStop(scanner.nextLine());
+        System.out.println("Podaj długość trasy w km : ");
+        route.setLength(scanner.nextInt());
+        System.out.println("Początek twojej trasy jest : " + route.getPlaceStart() + " , a koniec jest " + route.getPlaceStop());
+        System.out.println("Całkowita długość trasy to " + route.getLength());
+
+    }
+    public void routeLevelOfDifficulty (){
+        System.out.println("Podaj poziom trudności trasy :");
+        System.out.println("1 - Łatwy");
+        System.out.println("2 - Średni");
+        System.out.println("3 - Wymagający");
+        System.out.println("4 - Bardzo trudny");
+        int levelOfDifficulty = scanner.nextInt();
+        if (levelOfDifficulty == 1){
```

Comment:
tu może zrobić switcha z enuma, ale to tylko sugestia optymalizacyjna

Reply:
Potrzebna obsługa wyjątków